templates/meta-sharing.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Gotcha. In the interest of time, can I get you take care of this part? I don’t know my way around JetPack much at all.